gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -19224,6 +19368,262 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  uicc_clinical_m:
+    $ref:
+      - "_terms.yaml#/uicc_clinical_m/common"
+    enum:
+      - M0
+      - M1
+      - M1a
+      - M1b
+      - M1c
+      - MX
+      - cM0 (i+)
+      - Unknown
+      - Not Reported
+
+  uicc_clinical_n:
+    $ref:
+      - "_terms.yaml#/uicc_clinical_n/common"
+    enum:
+      - N0
+      - N0 (i+)
+      - N0 (i-)
+      - N0 (mol+)
+      - N0 (mol-)
+      - N1
+      - N1a
+      - N1b
+      - N1bI
+      - N1bII
+      - N1bIII
+      - N1bIV
+      - N1c
+      - N1mi
+      - N2
+      - N2a
+      - N2b
+      - N2c
+      - N3
+      - N3a
+      - N3b
+      - N3c
+      - N4
+      - NX
+      - Unknown
+      - Not Reported
+
+  uicc_clinical_stage:
+    $ref:
+      - "_terms.yaml#/uicc_clinical_stage/common"
+    enum:
+      - Stage 0
+      - Stage 0a
+      - Stage 0is
+      - Stage I
+      - Stage IA
+      - Stage IA1
+      - Stage IA2
+      - Stage IA3
+      - Stage IB
+      - Stage IB1
```

Comment:
the ajcc_clinical_stage enums also include a bunch related to cervix, e.g. Stage IB Cervix. These have not been included in the uicc_clinical_stage enums. guessing it's intentional?